lte/gateway/python/scripts/enodebd_cli.py

```diff
@@ -167,8 +167,14 @@ def get_enb_status(client, args):
     _print_str_status_line('GPS Longitude', enb_status.gps_longitude)
     _print_str_status_line('GPS Latitude', enb_status.gps_latitude)
     _print_str_status_line('FSM State', enb_status.fsm_state)
-
-
+    _print_str_status_line('GPS Aititude', enb_status.gps_altitude)
+    _print_str_status_line('Vendor', enb_status.vendor)
+    _print_str_status_line('Model Name', enb_status.model_name)
+    _print_prop_status_line('RF State', enb_status.rf_state)
+    _print_str_status_line('SW Version', enb_status.sw_version)
+    _print_str_status_line('Uptime', enb_status.uptime)
+  
```

Comment:
Please run the script on https://github.com/magma/magma/tree/master/lte/gateway/docker/python-precommit to fix all formatting issues.

Reply:
OK, I'll deal with it as soon as possible ，thank you!